src/plat/quartz64/config.cmake

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+cmake_minimum_required(VERSION 3.7.2)
+
+declare_platform(quartz64 KernelPlatformQuartz64 PLAT_QUARTZ64 KernelSel4ArchAarch64)
+
+if(KernelPlatformQuartz64)
+
+    declare_seL4_arch(aarch64)
+    set(KernelArmCortexA55 ON)
+    set(KernelArchArmV8a ON)
```

Comment:
The CortexA55 is an ArmV8.2 core.  The OdroidC4 was already added to the kernel as the first CortexA55 platform. At some point we should check whether the kernel should do any special handling for any extended features.